cerializer/compiler.py

```diff
@@ -0,0 +1,82 @@
+# pylint: disable=protected-access
+
+import distutils.core
+import hashlib
+import importlib.machinery
+import os.path
+import sys
+
+import Cython.Build.Dependencies
+import Cython.Build.Inline
+import Cython.Compiler.Main
+import Cython.Utils
+
+
+def compile_code(code):
+	return _cython_inline(code)
+
+
+def _load_dynamic(name, module_path):
+	return importlib.machinery.ExtensionFileLoader(name, module_path).load_module()
+
+
+def _cython_inline(complete_code, lib_dir = os.path.join(Cython.Utils.get_cython_cache_dir(), 'inline')):
```

Comment:
This function is definitely very interesting. Please write a docstring about everything this does, how and why. Also it would be nice to point out the original source.